### ISSUE TITLE
node-red-node-twitter support for favorites tweets

### ISSUE THREAD

**Issue by bigretromike** [node-red/node-red-nodes#497]
https://flows.nodered.org/node/node-red-node-twitter
"Tweets of who you follow will include their retweets and favourites"

Is it still possible to configure flow to get only favorites tweets for given user?

**Comment by knolleary** [comment]
I don't think so. The recent changes to the Twitter API have meant certain features in the node are no longer available. For this option, we use https://developer.twitter.com/en/docs/tweets/timelines/api-reference/get-statuses-home_timeline - which doesn't appear to mention favourites.

**Comment by bigretromike** [comment]
https://developer.twitter.com/en/docs/tweets/post-and-engage/api-reference/get-favorites-list

**Comment by knolleary** [comment]
It's possible - but our focus is on the core Node-RED project at the moment. 

The hard part, with all the recent API changes, is the rate-limiting. If this was added to one of the existing options, it would double the number of api requests we'd be making - and therefore half the rate at which we're able to poll the api.

The alternative would be to add it as an entirely new option on the node - that only sends favourites and not other events.

If you're waiting for us to do it, then I cannot commit to when that will happen. If you wanted to submit a pull-request, then it would certainly help.

**Comment by bigretromike** [comment]
rate-limit is a problem most of those request should be seperate object that count it for itself. If there was already working function that support those limit for twitter I could probably push something, but the node.js is not my strong side and I wouldn't want to push any bad-practice-code anyway. 

I will dig around, but probably wait for someone to pick that up, or just find alternative to this, as I wanted to use this for twitter related project. I pick node-red because that 'outdated' information that I find, play with it a bit and it was all ok. 

Thanks anyway to very quick respond ! Great project!
